src/index.html

```diff
@@ -64,9 +64,26 @@
         <footer class="footer">
           <section class="footer__marketing container">
             <h3>The ease of use and speed you've been looking for - the functionality and extensibility you didn't know you needed.</h3>
+              <ul class="footer__marketing-features">
+                <li>
+                  <img src="./images/features/aspect-ratio.svg" alt="aspect ratio icon">
+                </li>
+                <li>
+                  <img src="./images/features/gif.svg" alt="gif icon">
+                </li>
+                <li>
+                  <img src="./images/features/mic.svg" alt="microphone icon">
+                </li>
+                <li>
+                  <img src="./images/features/flash.svg" alt="flash icon">
+                </li>
+                <li>
+                  <img src="./images/features/extension.svg" alt="extension icon">
+                </li>
+              </ul>
             <div class="footer__marketing-details">
-              <a href="https://github.com/wulkano/kap" class="footer__marketing-download link--light is-bold">View and download on GitHub</a>
-              <span>OS X / macOS 10.10 or above required</span>
+              <a href="https://github.com/wulkano/kap" class="footer__marketing-download link--light is-bold">View and contribute on GitHub</a>
+              <span> macOS 10.10 or above required</span>
```

Comment:
So, should we just remove this?? Or should I put a small SVG in there? @skllcrn how important is this?